src/main/java/edu/byu/cs/autograder/test/PreviousPhasePassoffTestGrader.java

```diff
@@ -47,7 +48,7 @@ private <T> Set<T> allPreviousPhases(PhaseFunction<T> func) throws GradingExcept
             Phase previous = gradingContext.phase();
             while ((previous = PhaseUtils.getPreviousPhase(previous)) != null) {
                 RubricConfig rubricConfig = DaoService.getRubricConfigDao().getRubricConfig(previous);
-                if(rubricConfig.passoffTests() != null) {
+                if(rubricConfig.items().get(Rubric.RubricType.PASSOFF_TESTS) != null) {
```

Comment:
Note: `RubricConfig` is still a class. We could provide helpful methods to preserve the existing method calls, and avoid all these code updates.

Reply:
There are only two instances where we get this with a specific type in mind rather than a dynamic variable type. I would think that adding extra methods would add extra clutter for not that much added benefit. Do you disagree?